examples/game/lib/image_map.dart

```diff
@@ -15,7 +15,7 @@ class ImageMap {
   }
 
   Future<Image> _loadImage(String url) async {
-    Image image = await _bundle.loadImage(url);
+    Image image = await _bundle.loadImage(url).first;
```

Comment:
He's trying to wait for all the image to be loaded before starting the game.  There's more work to be done here to support animated images.